startupshutdown/startupshutdown.go

```diff
@@ -1,73 +1,76 @@
 package startupshutdown
 
 import (
-	"log"
-	"os"
-	"os/exec"
 	"strings"
 
+	"github.com/google/uuid"
```

Comment:
We added a library to generate a UUID for the request_id directive  

      "github.com/nu7hatch/gouuid"

can  we uses the same one  for what you want rather than add another dependency?  Or if the google one is better can we remove the nu7hatch library?

Reply:
I prefer the google library 😄

Reply:
Ok, once you have submitted this pr we can update the request_id package to use the google library also.